source/dialog/startup/startup_start.sqf

```diff
@@ -130,36 +130,79 @@ switch (_index) do {
     };
     case index_amount_zones_20: {
         zones_number = 20
-    };    
+    };
 };
 
 // Starting CP
 _index = lbCurSel 2103;
-switch (_index) do {
-    case index_amount_cp_60: {
+switch (_index) do
+{
+
+    case index_amount_cp_600:
```

Comment:
We have excess space above here and the case's here don't conform to the model we're going with. It's `index_amount_cp_200: {` for example. You'll probably have a merge conflict here as well though after #59 gets merged.